app/forms/flood_risk_engine/steps/partnership_form.rb

```diff
@@ -0,0 +1,46 @@
+module FloodRiskEngine
+  module Steps
+    class PartnershipForm < BaseForm
+      def self.factory(enrollment)
+        contact = Contact.new
+        new(contact, enrollment)
+      end
+
+      def self.params_key
+        :partnership
+      end
+
+      def self.max_length
+        70
+      end
+
+      # Used to work out whether this is the first partner being created.
+      # Needs to start at one for translations (:one and :other syntax)
+      def partner_count
+        enrollment.organisation.partners.count + 1
```

Comment:
The count is effectively, the number of partners persisted in the database, plus the 1 you are currently building. I'll update the comment to that effect.